src/libs/vtkh/compositing/CMakeLists.txt

```diff
@@ -98,7 +100,7 @@ if (MPI_FOUND)
 
     # build time only defs
     target_compile_definitions(vtkh_compositing_mpi PRIVATE ASCENT_EXPORTS_FLAG)
-    set_target_properties(vtkh_compositing_mpi PROPERTIES CXX_VISIBILITY_PRESET hidden)
+#    set_target_properties(vtkh_compositing_mpi PROPERTIES CXX_VISIBILITY_PRESET hidden)
```

Comment:
Looks good! one suggestion -- We need the conditional logic for the mpi variants of a few libs that were comment out in the first pass.

Reply:
Whoops! Let me see what that was... definitely a mistake on my part